### ISSUE TITLE
Original PAL cartridge 3 in 1 Super Mario Bros + Tetris + Nintendo World Cup issue

### ISSUE THREAD

**Issue by Alex-Catalin** [tjanas/Nt-Mini-Legacy-Jailbreak#5]
The issue with the game cartridge Super Mario Bros + Tetris + Nintendo World Cup, is that only Tetris will boot up of all 3 included titles. If I chose Super Mario Bros or Nintendo World Cup at the game selection screen, it will immediately return to the game selection screen and will not boot up the game.

Using both revisions of the rom on an Everdrive, I am unable to start the game at all, I get a black screen and that's that. 

Steps to reproduce the issue:

1. Update the firmware of the 2017 NT Mini to the latest jailbreak firmware version that is on this repository.
2. Insert the game cartridge in the console and boot it up
3. The game selection screen shows up with Super Mario Bros, Tetris and Nintendo World Cup
4. Try to start Super Mario Bros, the screen will flicker for a millisecond and we are back at the game selection menu from step 3.
5. Try to start Tetris and the game will start and play just fine.
6. Reset the game and go back to the game selection screen and select Nintendo World Cup, the same issue mentioned at step 4 will occur.

**Comment by tjanas** [comment]
Thank you for your bug report @Alex-Catalin. Have you contacted Analogue about this issue, especially since it occurs with a real cartridge?

**Comment by Alex-Catalin** [comment]
@tjanas I wonder if they will get back to me since my NT Mini is using the jailbreak firmware instead of the official one. I did not test what happens with the official firmware, but I don't want to go back to it due to some sound issues that it has.

Yesterday I tried to start the rom files from the SD card, using the jailbreak NES core and I got a black screen, just like on the Everdrive. This is exactly the same result that I got on my MiSTer and I opened a bug for their NES core as well. So, it appears that this specific rom is a bit more special and it is interesting how running the rom file (not working at all) behaves totally different vs when running the real cart (only Tetris boots up).

I will write to Analogue support as well. Thank you!
